dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -100,6 +101,10 @@ public final class ZenodoHelper {
     private static String zenodoUrl;
     private static String zenodoClientID;
     private static String zenodoClientSecret;
+    /**
+     * Comma-separated list of GitHub organization names and/or GitHub repo names, e.g., &qout;broadinstitute,DataBiosphere/topmed-workflow-variant-calling&quot;
```

Comment:
Separately, I think per repo is more complicated than needed. I think just by organization would be easier to document/communicate

Reply:
> Separately, I think per repo is more complicated than needed. I think just by organization would be easier to document/communicate

I'm fine with that, but I did add support for individual repos because your email mentioned allowing opt-out for specific workflows, so I had sorted covered that (down to the repo level, not workflow level).

Reply:
See below, I think similar issue.